### PR TITLE
Proofread readme

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Comment by greg0ire** [comment]
Wait there are more docs, I'm going to review them too

**Comment by TomasVotruba** [comment]
The smaller PR on md files the better. Usually people change to much, breaking main intent and feedback and comments drag so long so it ends up in the trash few months later.

**Comment by greg0ire** [comment]
Ok, I will make one separate PR for docs.

File: README.md [hunk 1]
```diff
@@ -440,39 +471,23 @@ Use [`create` command](/docs/RectorRecipe.md).
...
-    Do you need to fix coding standards? Run:
-
-    ```bash
-    composer fix-cs
-    ```
-
-We would be happy to accept PRs that follow these guidelines.
-
 ## Run Rector in Docker
 You can run Rector on your project using Docker:
```
Comment (TomasVotruba) [inline README.md]: There should be at least link to separated file

Comment (greg0ire) [reply README.md]: Done.

**Comment by greg0ire** [comment]
Tone the long lines commits down to just lines in the code to avoid horizontal scrolling.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Pull request merged**